pkg/authz/permission_list_app.go

```diff
@@ -73,4 +73,68 @@ var AppPermissionList = []Permission{
 		Name:          "primaza:app:leader-election",
 		Verbs:         []string{"create", "patch"},
 	},
+	{
```

Comment:
Wonder why all these are changed in this PR not related to permissions

Reply:
There was some bug with code generation. I tried to generate it manually.
I sent a separate PR #285 to fix the issue. Please take a look.